### PR TITLE
ORCA: parse CD spectrum (rotatory strengths)

### PR DESCRIPTION
**Pull request by berquist**
Closes https://github.com/cclib/cclib/issues/440.

### CODE REVIEW

File: cclib/parser/orcaparser.py [hunk 1]
```diff
@@ -959,6 +959,27 @@ def energy_intensity(line):
...
+        if line.strip() == "CD SPECTRUM":
+            # -------------------------------------------------------------------
+            #                              CD SPECTRUM
+            # -------------------------------------------------------------------
+            # State   Energy Wavelength       R         MX        MY        MZ   
+            #         (cm-1)   (nm)       (1e40*cgs)   (au)      (au)      (au)  
+            # -------------------------------------------------------------------
+            #    1   43167.6    231.7      0.00000   0.00000  -0.00000   0.00000
+            #
+            # Presumably the transition energies have already been parsed.
```
Comment (ATenderholt) [inline cclib/parser/orcaparser.py]: Should there be a warning if the energies haven't already been parsed? Or perhaps be more aggressive with an assert?

File: test/data/testTD.py [hunk 2]
```diff
@@ -185,6 +190,10 @@ def testsecs_transition(self):
         """ROCIS does not form singly excited configurations (secs)"""
         pass
 
+    def testrotatsnumber(self):
+        """ROCIS does not calculate rotatory strengths"""
```
Comment (langner) [inline test/data/testTD.py]: Is this specific to GAMESS?

Comment (berquist) [reply test/data/testTD.py]: Expand the context (that's a GitHub bug I think), it's specific to ORCA.

File: cclib/parser/orcaparser.py (hunk 1 above)
Comment (langner) [reply cclib/parser/orcaparser.py]: Did you want to do this Eric?

**Comment by langner** [comment]
I'm redirecting this to the `release-1.6.1` branch, so we can continue 1.6.2 things until we release 1.6.1. I'll merge into `master` from `release-1.6.1` as needed.

File: cclib/parser/orcaparser.py (hunk 1 above)
Comment (berquist) [reply cclib/parser/orcaparser.py]: It's here: https://github.com/cclib/cclib/pull/684/files#diff-8d648f11426b9efdbf41dbfe1b5135b9R987

**Pull request merged**